include/internal/biguint.h

```diff
@@ -10,7 +10,9 @@ class BigUInt
 
     explicit BigUInt(std::string value);
 
-    BigUInt operator*(BigUInt const &rhs);
+    BigUInt operator*(BigUInt const &rhs) const;
```

Comment:
They do not seem to be used now (sorry if mistaken). But indeed, can be useful to compute, for example, fees (in the future).

Reply:
Yes, not used in project, but useful for those who wish to use it.